en/docs/develop/hosting-authentication-endpoint-on-a-different-server.md

```diff
@@ -286,141 +201,98 @@ hosting accountrecoveryendpoint.war also on a separate web server.
-    . Now you can see that the authentication is redirected to:
-    `                       https://localhost:8443/authenticationendpoint/login.do                     `
+10. Start both Identity Server and tomcat and access `https://localhost:9443/dashboard`. Now you can see that the authentication is redirected to: `https://localhost:8443/authenticationendpoint/login.do`
 
     Now let’s take out account recovery endpoint into the external
     Tomcat server as well.
 
-11. Run `          setup-accountrecovery.sh         ` obtained from
-    [step 2](#HostingAuthenticationEndpointonaDifferentServer-step2) and
-    follow the instructions.
-12. Change the following section in
-    `           <TOMCAT_HOME>/webapps/authenticationendpoint/WEB-INF/web.xml          `
-    file and point to
-    `           IdentityManagementEndpointContextURL          `
-    into tomcat URL.
+11. Run `setup-accountrecovery.sh` obtained from [step 2](#HostingAuthenticationEndpointonaDifferentServer-step2) and follow the instructions.
+
+12. Change the following section in `<TOMCAT_HOME>/webapps/authenticationendpoint/WEB-INF/web.xml` file and point to `IdentityManagementEndpointContextURL` into tomcat URL.
 
     ``` xml
-        … 
-        <context-param>
-               <param-name>IdentityManagementEndpointContextURL</param-name>
-        <param-value>https://localhost:8443/accountrecoveryendpoint</param-value>
-           </context-param>
-        …
+    … 
+    <context-param>
+            <param-name>IdentityManagementEndpointContextURL</param-name>
+    <param-value>https://localhost:8443/accountrecoveryendpoint</param-value>
+        </context-param>
+    …
     ```
 
-13. In
-    `           <TOMCAT_HOME>/accountrecoveryendpoint/WEB-INF/classes/RecoveryEndpointConfig.properties          `
-    file, uncomment and change it to identity server.
+13. Open the `<TOMCAT_HOME>/accountrecoveryendpoint/WEB-INF/classes/RecoveryEndpointConfig.properties` file, uncomment the following config, and change it to `identity server`.
 
     ``` xml
-        identity.server.service.contextURL=https://localhost:9443/services/
+    identity.server.service.contextURL=https://localhost:9443/services/
     ```
 
-14. Uncomment and change the user portal reference in
-    `           <TOMCAT_HOME>/account          `
-    `           recovery          `
-    `           endpoint/WEB-INF/web.xml          `
+14. Uncomment and change the user portal reference in the `<TOMCAT_HOME>/accountrecoveryendpoint/WEB-INF/web.xml` file.
 
     ``` xml
-         …
-          <context-param>
-                <param-name>UserPortalUrl</param-name>
-                <param-value>https://localhost:9443/dashboard/index.jag</param-value>
-            </context-param>
-        ...
+    …
+    <context-param>
+        <param-name>UserPortalUrl</param-name>
+        <param-value>https://localhost:9443/dashboard/index.jag</param-value>
```

Comment:
dashboard is no longer there, change is to userportal URL

Reply:
is it correct to change this to:

```
<context-param>
        <param-name>MyAccountUrl</param-name>
        <param-value>https://localhost:9443/myaccount</param-value>
    </context-param>
```

Reply:
For 5.11.0 - 
```<param-name>MyAccountUrl</param-name>
        <param-value>https://localhost:9443/myaccount</param-value>```
For 5.10.0 -  
```<param-name>UserPortalUrl</param-name>
        <param-value>https://localhost:9443/user-portal</param-value>```